.github/workflows/test-rc-release.yaml

```
@@ -0,0 +1,152 @@
+---
+name: test-rc-release
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      rc_testing_branch:
+        # If a branch is given, the workflow will use it for deployment and testing.
+        # If no branch is provided, the workflow will create a new rc testing branch
+        # for deployment and testing.
+        description: |
+          rc_testing_branch: existing testing branch
+          (One and only one of rc_testing_branch and issue_url is required.)
+        required: false
+        default: ''
+      issue_url:
+        description: |
+          issue_url: the URL of the github issue that announce provider testing
+          (One and only one of rc_testing_branch and issue_url is required.)
+        required: false
+      base_git_rev:
+        description: 'The base git revision to test rc release'
+        required: false
+        type: string
+        default: 'main'
+
+jobs:
+  validate-manual-input:
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - name: Validate user input
+        if: ${{ (inputs.rc_testing_branch != '') == (inputs.issue_url != '') }}
```

```suggestion
        if: ${{ (inputs.rc_testing_branch != '') && (inputs.issue_url != '') }}
```
Should this be an AND? Equal operator seems confusing here.

---

This should be `==`. Only one of the value can be true

---

But if both are not set, it should exit as well

---

got it. 
But I think the expression should be explicit.
${{ (inputs. rc_testing_branch == '' && inputs. issue_url == '') || (inputs. rc_testing_branch != '' && inputs. issue_url != '') }}

---

Sure. I don't have a strong opinion on it. Just fix it in https://github.com/astronomer/astronomer-providers/pull/1135/commits/d65d8c1a7358f03170ce2ec1043714a2e0ce549b

---

thank you for your consideration and sorry for my strong opinion :)